app/views/posts/owed.haml

```diff
@@ -1,3 +1,22 @@
 - content_for :posts_title do
-  Threads You Owe Tags In
-= render partial: 'list', locals: {posts: @posts}
+  - if params[:view] == 'hidden'
+    Threads with Replies Owed Disabled
```

Comment:
Capital W, since we don't seem to be lowercasing prepositions in titles (e.g. "Threads You Owe Tags/Replies In")